features/products/new.feature

```diff
@@ -108,6 +108,22 @@ Scenario: Checking C2 Test updates Duplicate Records
   Then "C4 Test" checkbox should be enabled
   Then "C4 Test" checkbox should be unchecked
 
+Scenario: Checking CVU+ Product updates Bundle Options
```

Comment:
Could you add 1 more cucumber test where a product is saved

After 'the user fills out all product information with measures' (this step_definition will need to be created, but can leverage the user fills out all product information but measures)

User selects CVU+ 
User checks vendor and bundle patients
User selects Certification
User saves

make sure that vendor_patients is false